test/Microsoft.Extensions.DependencyModel.Tests/PackageResolverTest.cs

```diff
@@ -24,8 +24,8 @@ public void ShouldUseEnvironmentVariableToGetDefaultLocation()
                 .AddVariable("NUGET_PACKAGES", PackagesPath)
                 .Build();
 
-            var result = PackageCompilationAssemblyResolver.GetDefaultPackageDirectory(Platform.Unknown, environment);
-            result.Should().Be(PackagesPath);
+            var result = PackageCompilationAssemblyResolver.GetDefaultProbeDirectories(Platform.Unknown, environment);
```

Comment:
I'm not disagreeing.  I was actually making that same point.  This is a very common scenario and we should have tests that ensure it doesn't break.